server/src/main/java/org/elasticsearch/health/HealthStatus.java

```diff
@@ -34,6 +41,15 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeByte(value);
```

Comment:
we can use `writeEnum` and avoid the `byte` value we put in the enum 
(the enum ordinal serves this purpose already)

Reply:
I will adjust it in all places.